src/Marten/Events/Projections/ViewProjection.cs

```diff
@@ -109,6 +115,25 @@ public void Identity<TEvent>(Func<TEvent, TId> identityFunc)
             _groupers.Add(new SingleStreamGrouper<TId, TEvent>(identityFunc));
         }
 
+        /// <summary>
+        /// Use the TEvent event's stream key/id as the aggregate document id
+        /// </summary>
+        /// <typeparam name="TEvent"></typeparam>
+        /// <exception cref="InvalidOperationException"></exception>
+        public void Identity<TEvent>()
```

Comment:
We need to rename this method to make it clear that the identity is the stream identity